### PR TITLE
Add tests to cover CSV parsed options

### PR DESCRIPTION
**Pull request by yohanboniface**
@alexmojaki Something like this ? :)

### CODE REVIEW

**Comment by yohanboniface** [comment]
(Could restart the tests ? The error seems weird)

File: sandbox/grist/imports/import_csv_test.py [hunk 1]
```diff
@@ -62,8 +83,11 @@ def test_user_parse_options(self):
                                  "limit_rows": False, "quoting": 0, "start_with_row": 1,
                                  "delimiter": ",", "NUM_ROWS":10,
                                  "quotechar": "\"", "doublequote":True}}
-    parsed_file = import_csv.parse_file(_get_fixture('test_import_csv.csv'),
-                                        **options)[1][0]
+    options, parsed_file = import_csv.parse_file(_get_fixture('test_import_csv.csv'),
```
[the remark this answers is not in the document]
Comment (yohanboniface) [reply sandbox/grist/imports/import_csv_test.py:86]: Some of the options are not returned by `parse_file`:

```diff
  {'NUM_ROWS': 10,
   'delimiter': ',',
   'doublequote': True,
-  'escapechar': None,
   'include_col_names_as_headers': True,
-  'limit_rows': False,
   'lineterminator': '\n',
   'quotechar': '"',
-  'quoting': 0,
   'skipinitialspace': False,
   'start_with_row': 1}
```

So to check that passed options and returned options are the same, I need to drop those keys (plus `SCHEMA`, which is returned but not passed), which sounds a bit awkward. Do you think its worth it anyway ?

[the remark this answers is not in the document]
Comment (yohanboniface) [reply sandbox/grist/imports/import_csv_test.py]: OK, done then :)

**Comment by alexmojaki** [comment]
Thanks!

**Pull request merged**